packages/wonder-blocks-cell/src/components/internal/common.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+
+export const CellMeasurements = {
+    cellMinHeight: Spacing.xxLarge_48,
+
+    /**
+     * The cell wrapper's gap.
+     */
+    cellPadding: {
+        paddingVertical: Spacing.xSmall_8,
```

Comment:
@lillialexis I added the outer vertical padding here to match what we discussed here and in Figma.

Also, added an internal vertical spacing to the content/title wrapper to preserve the 14px for titles in case it's needed.